include/deal.II/base/vectorization.h

```diff
@@ -1224,6 +1240,26 @@ class VectorizedArray<double, 8>
   __m512d data;
 
 private:
+  /**
+   * Extract lower half of data field.
+   */
+  DEAL_II_ALWAYS_INLINE
+  __m256d
+  get_low() const
+  {
+    return _mm512_castpd512_pd256(data);
+  }
+
+  /**
+   * Extract higher half of data field.
+   */
+  DEAL_II_ALWAYS_INLINE
+  __m256d
+  get_high() const
```

Comment:
Regarding naming, is `high` the most comment name? I remember also reading `lower/upper`, so I wonder what the most common term is. As this is internal, I think picking whatever most closely resembles the Intel language is fine here.

Reply:
The Intel Developer Guide uses lower/upper so `get_lower()/get_upper()`?

Reply:
Yes.